pkg/server/routes.go

```diff
@@ -155,6 +155,7 @@ func (s *Server) setupRoutes() (err error) {
 			accounts.GET("/:accountID", s.AccountDetail)
 			accounts.PUT("/:accountID", csrf, s.UpdateAccount)
 			accounts.DELETE("/:accountID", csrf, s.DeleteAccount)
+			accounts.POST("/:accountID/change-password", csrf, s.ChangePassword)
```

Comment:
```suggestion
			accounts.POST("/:accountID/password", csrf, s.ChangePassword)
```

To keep the URL simpler